steps/xcode-build-version/0.0.1/step.yml

```diff
@@ -0,0 +1,66 @@
+title: Xcode Build Version
```

Comment:
It seems this step is about the Xcode project's version and build number, I would suggest a better title, like: `Get Xcode project version`